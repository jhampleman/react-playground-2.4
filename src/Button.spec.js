import React from 'react';
import { shallow } from 'enzyme';
import ButtonUI from './Button';

describe('ButtonUI', () => {
  let wrapper;
  beforeEach(() => wrapper = shallow(<ButtonUI />));
it('should render correctly', () => expect(wrapper).toMatchSnapshot());
  it('should render 1 RadioGroup', () => {
    expect(wrapper.find('RadioGroup').length).toEqual(1);
  });
  it('should render an H3 headline', () => {
  expect(wrapper.containsAllMatchingElements([
    <p id="myp" className="green">(This is in Button.js)</p>,
    <h3>Material UI - Test Elements Below</h3>
  ])).toEqual(true);
});
  it('should have aria label', () => {
    const ariatest = wrapper.find('#showme');
    expect(ariatest.props()).toHaveProperty('aria-label','Show and Hide the Forms');
    
  });

});