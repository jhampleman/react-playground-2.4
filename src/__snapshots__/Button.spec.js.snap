// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonUI should render correctly 1`] = `undefined`;
